lib/src/scss-shared/_nx-tag-variables.scss

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@use '../scss-shared/nx-dark-mode-helpers';
+
+%nx-tag-hover-style {
```

Comment:
Given that this group of variables do always have the same values, I am thinking putting them in `_nx-colors.scss` really is the best approach

Reply:
> Hmm for some reason, the variables --nx-selectable-color-dark and --nx-selectable-color-light are undefined when I assign them in _nx-colors.scss thinking

Ok, it appears that CSS variables values are intepreted in the scope of the selector where they were defined, not where they are used.  That's something we should keep in mind for the future.